### DISCUSSION TITLE
一个关于“搜索”功能的问题

### DISCUSSION THREAD

**Discussion by Marweey** [MaLuns/hexo-theme-async#73] [Q&A 疑难杂症] [answered]
您好！目前我的页面点击搜索按钮后表现为无反应，控制台会出现如下报错
![image](https://user-images.githubusercontent.com/97400065/225222780-827621b4-e403-4a89-8a89-0b289783da26.png)
![image](https://user-images.githubusercontent.com/97400065/225220088-ad09b3a9-5f70-4674-8572-ab9328c53ef5.png)
我使用的是本地搜索，大约三个月前第一次配置时可以正常使用，后也未修改过配置文件，只进行了主题版本更新，今日点击发现失效报错，重新按照您的使用指南进行了安装和配置也无效果，特来请问，盼复，感谢！

**Comment by MaLuns** [comment, marked as answer]
这应该是查询插件 js 文件没有加载进来，你的看下你配置文件里 plugin 是怎么配置的
